slave/server.go

```diff
@@ -1,25 +1,23 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
 	"net/http"
-)
 
-type Subtask struct {
-	Algorithm string
-	Salt      string
-	Digest    string
-	CharSet   string
-	Length    int
-	Start     string
-	End       string
-	Id        int
-}
+	"github.com/aaronang/cong-the-ripper/lib"
+)
 
-func subtaskHandler(w http.ResponseWriter, r *http.Request) {
-	// TODO
+func taskHandler(w http.ResponseWriter, r *http.Request) {
+	var t lib.Task
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&t); err != nil {
+		return
+	}
+	fmt.Println(t)
 }
 
 func main() {
-	http.HandleFunc("/", subtaskHandler)
-	http.ListenAndServe(":8080", nil)
+	http.HandleFunc(lib.CreateTaskPath, taskHandler)
+	http.ListenAndServe(lib.Port, nil)
```

Comment:
Port should be defined in the config, and have a default value of :8080 if not specified